console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx

```diff
@@ -198,7 +202,7 @@ export default function AddressSpacePage() {
     <PageSection variant={PageSectionVariants.light}>
       <Grid>
         <GridItem span={7}>
-          <AddressSpaceListFilterPage
```

Comment:
Earlier we discussed to rename it as AddressSpaceListToolbar

Reply:
mad changes for Toolbar and filter